### PR TITLE
Change list-content style

### PR DESCRIPTION
**Pull request by thelicato**
After a brief discussion yesterday (November 14 2022) I modified the style of the "list-content" div by adding a border-left instead of a full background and adding a brief description of the list.

Before: 
Screenshot from 2022-11-14 20-16-24
After:
Screenshot from 2022-11-14 20-21-29

### CODE REVIEW

**Comment by kolaente** [comment]
What does this look like with a list background_

**Comment by thelicato** [comment]
It looks like this:
Screenshot from 2022-11-15 10-34-03

File: src/components/list/partials/list-card.vue [hunk 1]
```diff
@@ -31,6 +31,7 @@
 			</BaseButton>
 
 			<div class="title">{{ list.title }}</div>
+			<div class="description">{{ list.description }}</div>
```
Comment (kolaente) [inline src/components/list/partials/list-card.vue:34]: I don't think showing the description is a good idea. If there's more than 2-3 lines (which can be very fast) there won't be enough space to show it. Combine that with a long list name and it gets worse.

File: src/components/list/partials/list-card.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
 			'has-background': blurHashUrl !== '' || background !== null,
 		}"
 		:style="{
-			'background-color': list.hexColor,
+			'border-left': `1rem solid ${list.hexColor}`,
```
Comment (kolaente) [inline src/components/list/partials/list-card.vue:8]: Please only set the border-color as an inline-style and move the rest to the stylesheet.

I also think 1rem is too much.

**Review by kolaente — CHANGES_REQUESTED** [review]

File: src/components/list/partials/list-card.vue (hunk 2 above)
Comment (thelicato) [reply src/components/list/partials/list-card.vue]: Ok, I left only the ``border-left-color`` as inline style. I also reduced the border to 0.8rem (even though I liked it more before)

**Review by kolaente — APPROVED** [review]

**Pull request merged**